packages/labs/test/popover2Tests.tsx

```diff
@@ -576,6 +577,97 @@ describe("<Popover2>", () => {
         });
     });
 
+    describe("deprecated prop shims", () => {
+        it("should convert position to placement", () => {
+            const popover = shallow(
+                <Popover2 inline={true} position={Position.BOTTOM_LEFT}>
+                    child
+                </Popover2>,
+            );
+            {
```

Comment:
whoa, nifty syntax trick

Reply:
what is this trick called? closures?

Reply:
Ok wow this is pretty cool! Makes the setup/assertion/setup/assertion pattern very clear.

But might this still be better accomplished with a helper function? Instead of repeating these 5 lines many times.
`assertPlacement(wrapper, expected)`

Reply:
it's called block scoping. you can read more about it here:
https://github.com/getify/You-Dont-Know-JS/blob/e342ec48f9e19c873e9a81dd36013c5875317dcc/es6%20%26%20beyond/ch2.md#block-scoped-declarations